### PR TITLE
Organized and added more cheats

### PR DESCRIPTION
**Pull request by yurirenjyo**

### CODE REVIEW

File: cht/Nintendo - Game Boy Advance/Pokemon - Emerald Version (USA, Europe) (Code Breaker).cht [hunk 1]
```diff
@@ -1,45 +1,598 @@
-cheats = 11 
+cheats = 150
+
+cheat0_desc = "Not all codes have been tested"
+cheat0_code = "Use at your own risk."
+cheat0_enable = false
```
Comment (RobLoach) [inline cht/Nintendo - Game Boy Advance/Pokemon - Emerald Version (USA, Europe) (Code Breaker).cht]: Don't think we need this warning.

File: cht/Nintendo - Game Boy Advance/Pokemon - Emerald Version (USA, Europe) (Code Breaker).cht [hunk 2]
```diff
@@ -1,45 +1,598 @@
-cheats = 11 
+cheats = 150
```
Comment (RobLoach) [inline cht/Nintendo - Game Boy Advance/Pokemon - Emerald Version (USA, Europe) (Code Breaker).cht]: Think this should be 149.

**Comment by RobLoach** [comment]
Thanks a lot!

**Comment by yurirenjyo** [comment]
image

I also noticed this in retroarch when I tried to see if the cheats came up properly. I probably should have done this first before anything. Looks like it doesn't want to work past cheat #99.

Sorry, I'm still pretty new to contributing on github.

**Pull request merged**